scarb/src/manifest_editor/add.rs

```diff
@@ -52,9 +52,20 @@ impl Op for AddDependency {
 
         let dep = Dep::resolve(*self, ctx)?;
 
+        let was_sorted = tab.as_table_like().map_or(true, |dep_table| {
```

Comment:
`get_table_mut` ensures `tab` is `TableLike`, so I think you could just `.unwrap()` here instead of doing `map_or`

Anyway, I think setting this to `false` in case it is not a table would be more correct -- we wouldn't want to touch this at all for sorting purpose

Reply:
This because I wanted to map the option directly, not sure how I can do it more concisely using `unwrap` tho. 

I was thinking because `get_table_mut` already ensure that it's `TableLike`, might as well just set the default to `true`, but yeah I think makes more sense to default to `false`.

Reply:
```rust
let was_sorted = {
    let values = tab.as_table_like().unwrap().get_values();
    is_sorted(values.iter().map(|(key, _)| key[0]))
};
```

Reply:
Wow I can so dumb sometimes. Alright will change it to that.